tests/foreman/ui/test_subscription.py

```diff
@@ -32,9 +32,9 @@
 pytestmark = [pytest.mark.run_in_one_thread, pytest.mark.skip_if_not_set('fake_manifest')]
 
 
-@pytest.fixture(scope='module')
-def golden_ticket_host_setup(function_entitlement_manifest_org, module_target_sat):
-    org = function_entitlement_manifest_org
+@pytest.fixture
+def golden_ticket_host_setup(function_sca_manifest_org, module_target_sat):
```

Comment:
Anyway we can get away from this name and switch it to something named more after SCA?

Reply:
Yeah, good point @chris1984. I just guess we would want to make the change in `master` first and back-propagate it to streams. The downside might be discontinuity in test results history, but that's probably not so important for a long failing test.